src/plugins/data/public/search/session/sessions_mgmt/components/actions/delete_button.tsx

```diff
@@ -49,7 +49,7 @@ const DeleteConfirm = (props: DeleteButtonProps & { onActionDismiss: OnActionDis
       onCancel={onActionDismiss}
       onConfirm={async () => {
         setIsLoading(true);
-        await api.sendCancel(id);
+        await api.sendDelete(id);
```

Comment:
we actually allow deleting only. 
cancelation is implemented server side, but can't be done from UI

Reply:
changing the name. because underneath it actually calls delete API